trezor-crypto/crypto/scrypt.c

```diff
@@ -25,6 +25,8 @@
  *
  * This file was originally written by Colin Percival as part of the Tarsnap
  * online backup system.
+ *
+ * This file was copied from https://github.com/Tarsnap/scrypt/blob/fbd5b105b75da42cbdf91ca55387724e312499b5/lib/crypto/crypto_scrypt-ref.c
```

Comment:
Let's use code from https://github.com/Tarsnap/scrypt/releases/tag/1.3.2? easier to track and diff later